src/components/riskmap/RiskMap.tsx

```diff
@@ -230,6 +230,12 @@ const RiskMap = ({
       });
       map.addLayer(accidentsLayerSegmentLayer as AnyLayer);
 
+      if (pointsVisible) {
```

Comment:
I think this logic already exists in the hook at line 334, so this should not be needed. In order to make the map initialization correct, you can just add `map` in the dependency error.